x-pack/plugins/threat_intelligence/public/modules/indicators/hooks/use_indicators.ts

```diff
@@ -134,6 +149,9 @@ export const useIndicators = ({
         )
         .subscribe({
           next: (response) => {
+            // eslint-disable-next-line no-console
+            console.info(response.rawResponse.hits.hits);
+
```

Comment:
shouldn't we remove this?

Reply:
good catch, thanks!

Reply:
I will add some more docs to display_name as well!